pkg/container/types/interval.go

```diff
@@ -202,3 +202,91 @@ func NormalizeInterval(s string, it IntervalType) (ret int64, rettype IntervalTy
 	}
 	return
 }
+
+func ParseIntervalToList(s string, it IntervalType, isDateAdd bool) (rets []int32, rettypes []IntervalType, err error) {
+	rets, err = parseInts(s)
+	// we need to solve the problem XXX_MicroSecond
```

Comment:
Are you sure this is the right behaviour?     Please take a look at parseInt function.    Why do you need this function again?

At least, if you are sure what your function is correct, fix the parseInt function.

Reply:
parseInts just parse "1 1:1:1:1.1" to slice[1, 1, 1, 1, 1, 1], I think I am right. why do you think I am wrong, please tell me the detail

Reply:
@fengttt  when I have "1.1" SECOND_MICROSECOND, the microsecond should be 100000 instead of 1.

Reply:
@fengttt do you think I will fix the parseint function or fix the NormalizeInterval function